services/src/messaging/src/main/java/org/openkilda/messaging/command/flow/InstallOneSwitchFlow.java

```diff
@@ -115,8 +116,9 @@ public InstallOneSwitchFlow(@JsonProperty(TRANSACTION_ID) final UUID transaction
                                 @JsonProperty("output_vlan_id") final Integer outputVlanId,
                                 @JsonProperty("output_vlan_type") final OutputVlanType outputVlanType,
                                 @JsonProperty("bandwidth") final Long bandwidth,
-                                @JsonProperty("meter_id") final Long meterId) {
-        super(transactionId, id, cookie, switchId, inputPort, outputPort);
+                                @JsonProperty("meter_id") final Long meterId,
+                                @JsonProperty("multi_table") final boolean multiTable) {
+        super(transactionId, id, cookie, switchId, inputPort, outputPort, multiTable);
```

Comment:
I think you forgot to update toString(), equals(), hashcode() and getters, setters